metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -88,19 +89,11 @@ class UnityCatalogSourceConfig(
         description="Name of the workspace. Default to deployment name present in workspace_url",
     )
 
-    only_ingest_assigned_metastore: bool = pydantic.Field(
-        default=False,
-        description=(
-            "Only ingest the workspace's currently assigned metastore. "
-            "Use if you only want to ingest one metastore and "
-            "do not want to grant your ingestion service account the admin role."
-        ),
+    _only_ingest_assigned_metastore_removed = pydantic_removed_field(
```

Comment:
This is actually the only mode supported, because `workspace_client` queries via the workspace, and there is only one metastore per workspace

Haven't 100% been able to test this because I don't have the AWS perms to create a second metastore and workspace :| but based on the databricks api docs, it doesn't seem like you can get catalogs from multiple metastores like we were trying to do

Reply:
Have confirmed this now. I think for us to support multiple metastores, we'll want to have the config take in a map from workspace url -> token

Reply:
I think you are right, this is what I remember as well.